recipes/libcurl/all/conanfile.py

```diff
@@ -225,35 +231,41 @@ def imports(self):
         #     but does not work on OS X 10.11 with SIP)
         # 2. copying dylib's to the build directory (fortunately works on OS X)
         if self.settings.os == "Macos":
-            self.copy("*.dylib*", dst=self._source_subfolder, keep_path=False)
+            self.copy("*.dylib*", dst=self.source_folder, keep_path=False)
```

Comment:
```suggestion
            copy(self, "*.dylib*", dst=self.source_folder, keep_path=False)
```
- Need to check the origin
from conan.tools.files import copy